introduction.md

```diff
@@ -1,8 +1,7 @@
-<<<<<<< HEAD
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?